plugins/ilert/src/constants.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const ILERT_INTEGRATION_KEY = 'ilert.com/integration-key';
```

Comment:
```suggestion
export const ILERT_INTEGRATION_KEY_ANNOTATION = 'ilert.com/integration-key';
```

perhaps? the other annotations use that form